user_modules/IsReady_Template/isReady.lua

```diff
@@ -1406,7 +1406,7 @@ end
 			--Start SDL
 			Test["Precondition_StartSDL_" ..tostring(TestCaseName) ] = function(self)
 
-				StartSDL(config.pathToSDL, config.ExitOnCrash)
+				StartSDL(commonPreconditions:GetPathToSDL(), config.ExitOnCrash)
```

Comment:
@dboltovskyi, I prefer to to use in all places the same call, in case of some future updates that may be caused.